spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -59,6 +59,58 @@
       end
     end
 
+    context "pagination" do
+      let!(:o1) {
+        create(:order_with_distributor, state: 'complete', shipment_state: 'ready',
+                                        completed_at: Time.zone.now )
+      }
+      let!(:o2) {
+        create(:order_with_distributor, state: 'complete', shipment_state: 'ready',
+                                        completed_at: Time.zone.now )
+      }
+
+      it "splits results according to line items from orders" do
+        15.times {
+          create(:line_item_with_shipment, order: o1)
+        }
+        5.times {
+          create(:line_item_with_shipment, order: o2)
+        }
+        o1.save!
+        o1.reload
+        o2.save!
+        o2.reload
+        visit_bulk_order_management
```

Comment:
I do have a quick suggestion regarding this block of code:

```diff
diff --git a/spec/system/admin/bulk_order_management_spec.rb b/spec/system/admin/bulk_order_management_spec.rb
index b80a77e15c..96250fa538 100644
--- a/spec/system/admin/bulk_order_management_spec.rb
+++ b/spec/system/admin/bulk_order_management_spec.rb
@@ -69,17 +69,16 @@ describe '
                                         completed_at: Time.zone.now )
       }
 
-      it "splits results according to line items from orders" do
+      before do
         15.times {
           create(:line_item_with_shipment, order: o1)
         }
         5.times {
           create(:line_item_with_shipment, order: o2)
         }
-        o1.save!
-        o1.reload
-        o2.save!
-        o2.reload
+      end
+
+      it "splits results according to line items from orders" do
         visit_bulk_order_management
 
         expect(page).to have_select2 "autogen4", selected: "15 per page" # should be default option
```

Reply:
Thanks JB. I thought it would be necessary to update the order, in this case; indeed it is not.

I've addressed this change and rebased :+1: